modules/terraform/Makefile

```diff
@@ -1,6 +1,6 @@
 TMP ?= /tmp
 TERRAFORM ?= $(BUILD_HARNESS_PATH)/vendor/terraform
-TERRAFORM_VERSION ?= 0.12.26
+TERRAFORM_VERSION ?= 1.0.11
```

Comment:
Would this break some module tests ? My concern is regarding tests that use older submodules with references to `list()` and `map()` instead of `tolist()` and `tomap()` and possibly other things we may not anticipate.

Reply:
This is only the default Terraform version. All our testing sets the Terraform version more explicitly.